containers/daap-athena-load/src/var/task/athena_load_handler.py

```diff
@@ -41,6 +45,23 @@ def handler(
         extraction.path, data_product_element, logger=logger
     )
 
+    try:
+        table_schema = (
```

Comment:
what i've suggested does mean we might need to add another attribute to `DataProductSchema` or at least have it populate DatabaseName with the data_product_name+version suffix

could make sure that key's value in the schema includes the suffix in `DataProductSchema.convert_schema_to_glue_table_input_csv()`

Reply:
Good point.

I've added `database_name` to the `BaseJsonSchema` as it can be calculated there. It's then inherited into `DataProductMetadata` and `DataProductSchema` classes. 

This is used in the convert method for `DatabaseName`